src/login/logind-button.c

```diff
@@ -155,7 +188,10 @@ static int button_dispatch(sd_event_source *s, int fd, uint32_t revents, void *u
                                    LOG_MESSAGE("Reboot key pressed."),
                                    "MESSAGE_ID=" SD_MESSAGE_REBOOT_KEY_STR);
```

Comment:
i think the log message should say what is going on here, i.e. should be different for the two cases of the if check below, and say what is happening.